openedxscorm/scormxblock.py

```diff
@@ -517,6 +538,8 @@ def update_package_fields(self):
         schemaversion = root.find(
             "{prefix}metadata/{prefix}schemaversion".format(prefix=prefix)
         )
+        
+        self.extract_navigation_titles(root, prefix)
```

Comment:
can if extract navigation only if `self.enable_navigation_menu` is set to `True`?

Reply:
The problem with this was that we are only updating package_fields when we don't upload a new file in `studio_submit()`. Therefore, if a course author changes the `enable_navigation_menu` field after he has uploaded the zip file and submitted once, it won't go into the `update_package_fields` function at all and the titles won't be extracted.